rust-webhook-server/loggerServer.js

```diff
@@ -0,0 +1,25 @@
+const http = require('http');
+const server = http.createServer();
+
+server
+  .on('request', (request, response) => {
+    let body = [];
+    request
+      .on('data', (chunk) => {
+        body.push(chunk);
+      })
+      .on('end', () => {
+        body = Buffer.concat(body).toString();
+
+        console.log(`==== ${request.method} ${request.url}`);
+        console.log('> Headers');
+        console.log(request.headers);
+
+        console.log('> Body');
+        console.log(body);
```

Comment:
nit: for readability
```suggestion
        console.log(JSON.stringify(body, (k,v) => v, 2);
```